src/styles/variables.scss

```diff
@@ -1,3 +1,5 @@
 // TUTAJ DODAJEMY ZMIENNE KOLORÓW
-
-// ex. $color-font--primary: #ffff;
+$color-button--primary: rgba(255, 0, 0, 0.8);
```

Comment:
Nie wiem czy to dobra nazwa bo primary dla tego komponentu nie jest primary dla innego :)

Reply:
$color-buttonPlay--primary @miron54, @Natal1a może coś w tym stylu?

Reply:
A ten kolor będzie wykorzystywany gdzieś jeszcze czy tylko jako tło dla tego elementu domyślnie? Bo jak gdzie indziej to przydałaby się zmienna nie sugerująca tego obiektu a jak tylko dla niego to po co w ogóle robić zmienną?

Reply:
Dobra uwaga z tą nazwą.
Dodałam zmienną żeby trzymać wszystkie kolory w tym samym miejscu, bo jak przypiszę kolor tylko w stylach do tego komponentu, to np przy zmianie kolorów może być problem z szukaniem gdzie co jest. Ja bym zostawiła ten kolor w zmiennej i może faktycznie nazwała "$color-buttonPlay--primary", ale dajcie jeszcze znać jeśli macie inne zdanie.